explainer-server/public/stylesheets/scss/layout/_toolbar.scss

```diff
@@ -163,16 +163,40 @@ $toolbarHeight: 50px;
   }
 
   &--loading {
+    .publication-state__icon
     .save-state__icon {
       visibility: visible;
     }
 
+    .publication-state__message
     .save-state__message {
       visibility: hidden;
     }
   }
 }
 
+.publication-state {
```

Comment:
As these are all the same as the `.save-state` - just adjust the name to be more generic and reuse the styles across both.